### PR TITLE
Add user-property assertion to signup

### PR DESCRIPTION
**Pull request by ovekyc**
ASSERT! and default email

### CODE REVIEW

**Comment by festinalente91** [comment]
other than lgtm

File: src/server/users/models.js [hunk 1]
```diff
@@ -77,6 +79,12 @@ const UserManager = {
       })
       .then(profileImgUrl => {
         userValue.profileImgUrl = profileImgUrl;
+
+        assert(userValue.name, 'Invalid proerty - name');
+        assert(userValue.facebookId, 'Invalid proerty - facebook id');
+        assert(userValue.profileImgUrl, 'Invalid proerty - profile image url');
+        assert(userValue.email, 'Invalid proerty - profile image url');
```
Comment (festinalente91) [inline src/server/users/models.js]: shouldn't it be facebook email?

Comment (ovekyc) [inline src/server/users/models.js]: I dont think so.
check out code line 69

**Comment by festinalente91** [comment]
lgtm

**Review by hewonjeong — APPROVED** [review]
LGTM

**Pull request merged**